gadopt/level_set_tools.py

```diff
@@ -77,35 +77,48 @@ def assign_level_set_values(
     epsilon: float | fd.Function,
     /,
     interface_geometry: str,
-    interface_coordinates: list[list[float]] | list[list[float], float] | None = None,
     *,
+    interface: sl.LineString | sl.Polygon | None = None,
+    interface_coordinates: list[list[float]] | list[list[float], float] | None = None,
```

Comment:
```suggestion
    interface_coordinates: list[list[float]] | tuple[float, float] | None = None,
```
If I've read your docs right, this should be the type annotation for the (centre, radius) case for `interface_coordinates`.